skills/azure-cost-calculator/references/services/databases/database-for-mysql.md

```
@@ -3,80 +3,98 @@ serviceName: Azure Database for MySQL
+| `skuName`     | Per-vCore: `'vCore'`; Business Critical: `'1 vCore'`; fixed tiers: SKU  | `'vCore'`, `'1 vCore'`, `'Standard_B4ms'`   |
+| `meterName`   | Compute, storage, IOPS, backup, or add-on meter                         | `'vCore'`, `'Additional IOPS'`, `'B4MS'`    |
 
-| Meter                            | skuName              | unitOfMeasure  | Notes                                                    |
-| -------------------------------- | -------------------- | -------------- | -------------------------------------------------------- |
-| `vCore`                          | `vCore`              | `1 Hour`       | Per-vCore rate, multiply via InstanceCount (GP/MO v5/v6) |
-| `B4MS`                           | `Standard_B4ms`      | `1 Hour`       | Fixed Burstable SKU, includes vCPU+RAM                   |
-| `8 vCore`                        | `Standard_E8d_v5`    | `1 Hour`       | Fixed MO Edsv5 size, includes vCPU+RAM                   |
-| `Storage Data Stored`            | `Storage`            | `1 GB/Month`   | Standard LRS data storage                                |
-| `Backup Storage LRS Data Stored` | `Backup Storage LRS` | `1 GB/Month`   | Backup storage (LRS); ZRS also available                 |
+## Meter Names
+| Meter                                  | skuName              | unitOfMeasure | Notes                                  |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)